Feature/us 014 at sce api container

ATdllerenas opened the pull request:
Here is the at-sce-api Container, The second commit was the modification of the packages  aren't seen inside the container, the third commit was the change from UserRepository Class to UserRepository Interface and the same with UserServices. I didn't know the summary was the update of each change.

Commit:
at-sce-api_Container

Commit:
at-sce-api_Container

Commit:
at-sce-api_Container

luisgalindoluna reviewed (approved):
Looks good

EmiliaGuillen-dev reviewed (approved):
looks good

ivanflores15 commented on at-sce-api/src/test/java/com/example/atsceapi/AtSceApiApplicationTests.java:
```diff
@@ -0,0 +1,13 @@
+package com.example.atsceapi;
```
Why package name is com.example?

ivanflores15 reviewed (commented):
@luisgalindoluna I think all the folder under .mvn/** should not be included, should be?

luisgalindoluna commented:
> @luisgalindoluna I think all the folder under .mvn/** should not be included, should be?

@ivanflores15 It shouldn't? As far as I could tell those files were created automatically because of spring init, I don't think they should cause any problems.

ivanflores15 reviewed (commented):
The convention for package naming is use lowercase, that means name of package should be com.example.atsceapi.**service** instead of com.example.atsceapi.**Service**

Commit:
Change of Extension and Capital Letters

-Change from Example Extension to Agilethought
-Change of capital letters to packages

Commit:
Changes in gitignore

-Added 2 lines to hide mvnw and mvnw.cmd

Commit:
Changes in gitignore and Delete files

-mvnw and mvnw.cmd removed

Commit:
Changes to ATSceApiApplication

AtSceApiApplication to ATSceApiApplication
